tracee: 0.9.2 -> 0.10.0

06kellyjac opened the pull request:
###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

Bump tracee to 0.10.0

Diff: https://github.com/aquasecurity/tracee/compare/v0.9.2...v0.10.0

Changelog: https://github.com/aquasecurity/tracee/releases/tag/v0.10.0

Fix integration tests after recent changes

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

06kellyjac commented:
Might need to also add GPLv2+ to the license

https://github.com/aquasecurity/tracee/issues/2588

Artturin commented:
ci failed

```
error: builder for '/nix/store/hdmhzmxb67h3qdx6akr775n80b9wdnxp-tracee-integration-0.10.0.drv' failed with exit code 1;
       last 10 log lines:
       > Hunk #6 succeeded at 317 (offset 3 lines).
       > Hunk #7 succeeded at 377 (offset 3 lines).
       > Hunk #8 succeeded at 385 (offset 3 lines).
       > Hunk #9 succeeded at 410 (offset 3 lines).
       > Hunk #10 succeeded at 431 (offset 3 lines).
       > Hunk #11 succeeded at 690 (offset 71 lines).
       > applying patch /nix/store/p6mdp31va661r7z8yizhcs52w7wx2wcp-test-EventFilters-prefix-nix-friendly.patch
       > patching file tests/integration/integration_test.go
       > Hunk #1 FAILED at 246.
       > 1 out of 1 hunk FAILED -- saving rejects to file tests/integration/integration_test.go.rej
       For full logs, run 'nix log /nix/store/hdmhzmxb67h3qdx6akr775n80b9wdnxp-tracee-integration-0.10.0.drv'.
       ```

06kellyjac committed:
tracee: 0.9.2 -> 0.10.0

Diff: https://github.com/aquasecurity/tracee/compare/v0.9.2...v0.10.0

Changelog: https://github.com/aquasecurity/tracee/releases/tag/v0.10.0

Fix integration tests after recent changes

06kellyjac commented:
thanks @Artturin, resolved
